includes/Checker/Runtime_Environment_Setup.php

```diff
@@ -48,13 +48,18 @@ static function () use ( $permalink_structure ) {
 				}
 			);
 
+			// Do not send post-install notification email, see https://github.com/WordPress/plugin-check/issues/424.
+			add_filter( 'pre_wp_mail', '__return_false' );
+
 			wp_install(
 				'Plugin Check',
 				'plugincheck',
 				'demo@plugincheck.test',
 				false
 			);
 
+			remove_filter( 'pre_wp_mail', '__return_false' );
```

Comment:
Would this be safer in case there is an existing plugin that is already disabling mail?

```suggestion
			$return_false = static function (): bool {
				return false;
			};
			add_filter( 'pre_wp_mail', $return_false );

			wp_install(
				'Plugin Check',
				'plugincheck',
				'demo@plugincheck.test',
				false
			);

			remove_filter( 'pre_wp_mail', $return_false );
```

Reply:
Since this runs only for the duration of a runtime check, not sure if there's even any risk associated with that. So while I understand the point, it seems a bit too defensive